blazor/getting-started/razor-class-library.md

```diff
@@ -185,18 +183,17 @@ You can create a **Blazor Web App** or **Blazor Server App** or **Blazor WebAsse
     ```
 
 
-3. For Blazor Web App, include the stylesheet reference in the `<head>` section and the script reference at the end of the `<body>` in the **~/Components/App.razor** file:
+3. Include Stylesheet and Script References Based on Project Type:
```

Comment:
Why we change this @Athiswaran-SF3980 ? we have to mention refer scripts in body section and css in head section